src/vcf/meta_entry.cpp

```diff
@@ -214,6 +214,13 @@ namespace ebi
             value["Type"] != "String") {
             throw new MetaSectionError{entry.line, "INFO metadata Type is not a Integer, Float, Flag, Character or String"};
         }
+
+        // Check for predefined tag AA
+        if (value["ID"] == "AA") {
+            if (value["Type"] != "String") {
```

Comment:
This block could be made more generic by using a `map<string, pair<string, string>>` which could be declared at namespace level. The keys would be IDss, and the values would be pairs with the Type and the message to display in case of error.